src/controllers/TodoController.js

```diff
@@ -48,6 +48,16 @@ class TodoController {
       res.status(500).json(error.message);
     }
   }
+
+  static async deleteSome(req, res) {
+    const { condition } = req.body;
+    try {
+      const response = await TodoService.deleteSome(condition);
```

Comment:
Not recommended to do like this when we getting condition from external side and providing directly to DB without preparing and checking. Lets make it specified method to require todo ids like in complete all method